server/models/connections.go

```diff
@@ -21,6 +24,17 @@ const (
 	NOTFOUND     ConnectionStatus = "not found"
 )
 
+const (
+	PLATFORM      string = "platform"
```

Comment:
@MUzairS15, two questions: 1) was this generated from schema? 2) can we avoid hardcoding and redefining these enums here? We already have these values defined in the schema. How do we avoid redefining them here?